sdk/attestation/azure-security-attestation/inc/azure/attestation/attestation_client.hpp

```diff
@@ -184,7 +184,7 @@ namespace Azure { namespace Security { namespace Attestation {
      * specified service instance.
      */
     Response<Models::OpenIdMetadata> GetOpenIdMetadata(
-        Azure::Core::Context const& context = Azure::Core::Context::ApplicationContext) const;
+        Azure::Core::Context const& context = {}) const;
```

Comment:
I thought I did.

Reply:
I meant, breaking change in Attestation. We are changing the default, so the signature changed, right?

Reply:
Non-snarky question: In what way would this change break a customer?

There is a theoretical break where a customer took a dependency on this method using ApplicationContext as the default and then expected to be able to cancel ApplicationContext. 

But we've already asserted that customers don't really interact with Context objects in general, so describing this break as anything other than extremely theoretical is a challenge.

And that is *especially* true since we basically have no evidence that there are any customers using the Attestation client.

Reply:
That's totally fair, and I support making that judgement call.

I wanted to make sure my understanding is correct, that changing a default parameter's default value, _in principle_, is breaking. Thanks for clarifying.